hunts/views.py

```diff
@@ -35,7 +35,14 @@ def index(request):
     if request.method == "POST":
         form = HuntForm(request.POST)
         if form.is_valid():
-            hunt = Hunt(name=form.cleaned_data["name"], url=form.cleaned_data["url"])
+            hunt = Hunt(
+                name=form.cleaned_data["name"],
+                url=form.cleaned_data["url"],
+                start_time=form.cleaned_data["start_time"],
+                end_time=form.cleaned_data["end_time"],
+            )
+            hunt.save()
```

Comment:
See my comment above about overriding save() instead of clean(). You can also connect to the `pre_save` signal as well.

https://stackoverflow.com/questions/4380879/django-model-field-default-based-off-another-field-in-same-model